test/controllers/patients_controller_test.rb

```diff
@@ -123,4 +123,30 @@ class PatientsControllerTest < ActionController::TestCase
       assert_redirected_to root_path
     end
   end
+
+  #confirm get :data_entry returns a success code
+  describe `data_entry method` do
```

Comment:
kind of a nit, but to stick with the style can we switch this out to single quotes instead of backticks?